lib/watchOrServe.js

```diff
@@ -6,16 +6,27 @@ module.exports = function ( gobblefile, getTask ) {
 		resuming;
 
 	resume();
+logger.info({message: 'gobblefile is ' + gobblefile});
+	function watch () {
+		pathwatcher.watch( gobblefile, type => {
 
-	pathwatcher.watch( gobblefile, type => {
-		if ( type === 'change' ) restart();
-	});
+			if ( type === 'change' ) {
+				logger.info({ code: 'GOBBLEFILE_CHANGED' });
+				return restart();
+			}
+
+			if ( type === 'delete' ) {
+				logger.info({ code: 'GOBBLEFILE_DELETED' });
+				return setTimeout( watch, 2000 );
```

Comment:
what about a `setInterval` that gets cancelled once the file exists again? Or, equivalently...

```js
function checkGobbleFileExists () {
  fs.exists( gobblefile, exists => exists ? restart() : setTimeout( checkGobbleFileExists, 100 ) );
}

// ...
if ( type === 'delete' ) {
  logger.info({ code: 'GOBBLEFILE_DELETED' });
  return setTimeout( checkGobbleFileExists, 100 );
}
```

That way you don't have to wait 2 seconds each time the editor saves. Thoughts?